### PR TITLE
Tidy up the python APIs

### PR DESCRIPTION
**Pull request by miratepuffin**
### What changes were proposed in this pull request?
Tidied up the python APIs to bring them more in line with what is available in rust and find missing features which need to be worked on. These include:

* Adding a `graph.latest()` until we can fix the PyRef issue discovered.
*

### CODE REVIEW

**Comment by Haaroon** [comment]
Fix your tests

File: raphtory/src/graph.rs [hunk 1]
```diff
@@ -36,18 +36,67 @@ impl Graph {
...
+            self.graph.add_edge(
+                t,
+                src,
+                dst,
+                &props
+                    .into_iter()
+                    .map(|f| (f.0.clone(), f.1.into()))
+                    .collect::<Vec<(String, dbc::Prop)>>(),
+            )
+        } else {
```
Comment (Haaroon) [inline raphtory/src/graph.rs:87]: Remove panic

Comment (Haaroon) [reply raphtory/src/graph.rs:87]: wait a second this looks like my code, thanks i forgot to expose it

File: raphtory/src/graph.rs [hunk 2]
```diff
@@ -146,55 +191,7 @@ impl Graph {
         }
     }
 
-    pub fn add_vertex(&self, t: i64, v: &PyAny, props: HashMap<String, Prop>) {
```
Comment (Haaroon) [inline raphtory/src/graph.rs:149]: why is this being removed to window graph?

**Review by Haaroon — CHANGES_REQUESTED** [review]

File: raphtory/src/graph.rs (hunk 2 above)
Comment (miratepuffin) [reply raphtory/src/graph.rs:149]: I just moved them around and added some comments - GitHub just being a little wonky

**Pull request merged**